justfile

```diff
@@ -142,25 +149,50 @@ lint: clippy fmt-check
 
 package-dir:
     mkdir -p packages
+    rm -rf packages/prep
 
-[linux]
-package: build get-binary package-dir
-    tar cv {{output-filename}} | gzip -9 > "packages/cargo-binstall-{{target}}.tgz"
+package-prepare: build package-dir
+    just get-binary packages/prep
+    -just get-debuginfo {{output-filename}} packages/prep
+
+    just get-output detect-wasi{{output-ext}} packages/prep
```

Comment:
Why do we fetch output of `detect-wasi` though?

Reply:
could be useful. it's a "full" package after all